HackIllinois.xcodeproj/project.pbxproj

```diff
@@ -199,6 +201,13 @@
 /* End PBXFileReference section */
 
 /* Begin PBXFrameworksBuildPhase section */
+		29D333FC7383657348963D23 /* Frameworks */ = {
```

Comment:
Are the changes in the Xcode project other than the addition of UIViewExtensions intentional, if so what are they?

Reply:
I don't think I changed anything here, what would this come from?

Reply:
everything seems okay yes